### ISSUE TITLE
Some of the messages that are received are not shown on the dashboard.

### ISSUE THREAD

**Issue by Galactic21** [msztolcman/sendria#16]
Good afternoon, I wanted to expose a problem that may be beyond my knowledge but I am using Sendria in a DEV environment and what is happening is that the messages I want to send to Sendria seem to be well received but they are not being displayed on the dashboard. In this image we can see messages being accepted on 08/29/2022. 

<img width="1102" alt="image" src="https://user-images.githubusercontent.com/32776046/187246721-dc810229-4404-4387-8a86-c273a3b3742e.png">

However, in the following image, we can see that the last message shown is from 08/26/2022. 

<img width="1911" alt="Screenshot at Aug 29 17-15-51" src="https://user-images.githubusercontent.com/32776046/187247417-3330eefa-bf4d-416e-8992-556418f60b55.png">

The database file contains about 13MB so it has nothing to do with the limit of a sqlite database. And the last time a file was saved in the database is also 08/26/2022.

<img width="432" alt="image" src="https://user-images.githubusercontent.com/32776046/187247506-38a9e841-d879-449d-af81-30d7b5ef86a3.png">

In conclusion, messages are being received in the Sendria logs but not being saved in the sqlite file.

I ask for help in this matter if possible.
Thank you.
